apps/builder/app/env/vite-env.d.ts

```diff
@@ -0,0 +1,4 @@
+interface ImportMetaEnv {
```

Comment:
can this interface be defined at env.static.server?

Reply:
Error in non d.ts files
<img width="502" alt="image" src="https://github.com/webstudio-is/webstudio/assets/5077042/49718dbf-1c76-4e6a-bbe5-fc5ba319bd24">

Reply:
<img width="532" alt="image" src="https://github.com/webstudio-is/webstudio/assets/5077042/6f709d9d-7fd2-4389-a259-0a78292e18a4">